com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/analytics/collectors/AnalyticsCollector.java

```diff
@@ -70,7 +70,7 @@ public AnalyticsData collectAnalyticsData(DelegateExecution context) {
 
     }
 
-    public AbstractCommonProcessAttributesCollector<?> getProcessType(ProcessType processType) {
+    public AbstractCommonProcessAttributesCollector getProcessType(ProcessType processType) {

```

Comment:
Isn't there a compiler warning now about usage of raw types?

Reply:
I don't see one in my ide